play/src/containers/GameSessionContainer.tsx

```diff
@@ -74,6 +73,15 @@ export default function GameSessionContainer({ apiClient }: GameSessionContainer
     }
   };
 
+  const updateTeamScore = async (inputTeamId: string, inputScore: number) => {
+    try {
+      await apiClient.updateTeam({id: inputTeamId, score: inputScore});
+    }
+    catch (error) {
+      console.error(error)
```

Comment:
Should we re-throw the error the it can be captured by the UI component? That way UI can decide to ignore or show a proper error message to the user

Reply:
this makes a ton of sense. I'm going to write this into the next PR, once the API calls are consolidated into the proper `Pregame` and `GameInProgress` containers